src/cpp/rtps/resources/TimedEventImpl.h

```diff
@@ -108,7 +73,8 @@ class TimedEventImpl
      * @param time New expiration time.
      * @return true on success
      */
-    bool update_interval(const Duration_t& time);
+    bool update_interval(
+            const Duration_t& time);
```

Comment:
Rename param to `interval` (in the doc also). The doc mentions `restart_timer`, which does not exist on this class.